"""Raised when a number is expected, but the provided value is invalid."""
from illud.exception import IlludException


class InvalidNumberException(IlludException):
    """Raised when a number is expected, but the provided value is invalid."""
